node/nodeRunner.go

```diff
@@ -676,23 +679,23 @@ func (nr *nodeRunner) logSessionInformation(
 	sessionInfoFileOutput string,
 	managedCoreComponents mainFactory.CoreComponentsHandler,
 ) {
-	configs := nr.configs
 	statsFolder := filepath.Join(workingDir, core.DefaultStatsPath)
+	fileNamesConfig := nr.configs.ConfigurationFileName
```

Comment:
this variable should become `configurationPaths`

Reply:
done (everywhere)